src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCount = 40;
+  const totalSum = dailyRentalCount * days;
+  const discountFromTreeDays = 20;
```

Comment:
Please ensure that the variable names are descriptive and correctly spelled. The name `discountFromTreeDays` seems to be a typo and should rather be `discountFromThreeDays` to accurately reflect its purpose, as per the checklist item 1.